layer/ags_capture/dll_main.cpp

```diff
@@ -115,6 +115,27 @@ EXTERN_C AGSReturnCode AMD_AGS_API gfxrecon_agsInitialize(int
     return AGS_FAILURE;
 }
 
+// When this API function is called, it means the app is trying to use AGS v5.x.x versions
+EXTERN_C AGSReturnCode AMD_AGS_API gfxrecon_agsInit(AGSContext**            context,
+                                                    const AGSConfiguration* config,
+                                                    AGSGPUInfo*             gpuInfo)
+{
+    if (gfxrecon::Initialize())
+    {
+        GetDispatchTable().agsInitNotSupported();
+    }
```

Comment:
Is `gfxrecon::Initialize()` and `AgsDispatchTable::agsInitNotSupported();` required, or can the error message be logged here directly and the new dispatch table entry `AgsDispatchTable::agsInitNotSupported();` be removed?

Reply:
Rodrigo asked the same question. Here is my answer: "I tested. If put the log code here, it won't do the log, actually doing nothing. To make it doing log here, I have to add logging setup solely for this module, reading environment parameters and calling log::init(settings) and all that. I think this way is cleaner."

Reply:
I see. Does it keep the desired logging behavior if the dispatch table entry `agsInitNotSupported` is removed and this code is changed to this:
```
if (gfxrecon::Initialize())
{
    GFXRECON_LOG_ERROR_ONCE("AGS version mismatch for GFXR capture: GFXR doesn't support this version of "
                           "amd_ags_x64.dll. Please remove the capture DLL and rename the original DLL back.");
}
```
The reason I ask is because if the `agsInitNotSupported` table entry is not required, removing it cleans up the code in this file.

Reply:
If we change the code only to: 
```
if (gfxrecon::Initialize())
{
    GFXRECON_LOG_ERROR_ONCE("AGS version mismatch for GFXR capture: GFXR doesn't support this version of "
                           "amd_ags_x64.dll. Please remove the capture DLL and rename the original DLL back.");
} 
```, 
it won't log anything. We have to add logging setup code in this file to make it log. If we set up logging at this proxy API point, in this file, we can remove `agsInitNotSupported`. I looked at other proxy API dlls, there is no logging code. I thought the original GFXR designer meant to keep all the logging code at the encode level. So, now I am not sure which way is better. Both ways have pro and con.

Reply:
I see, thanks for the explanation--I was thinking that logging would be setup with `gfxrecon::Initialize`. I think this is OK as-is.